src/crawlee/crawlers/_playwright/_playwright_crawler.py

```diff
@@ -494,7 +495,8 @@ class _PlaywrightCrawlerAdditionalOptions(TypedDict):
     """A `BrowserPool` instance to be used for launching the browsers and getting pages."""
 
     browser_type: NotRequired[BrowserType]
-    """The type of browser to launch ('chromium', 'firefox', or 'webkit').
+    """The type of browser to launch ('chromium', 'firefox', 'webkit' or 'chrome'). Use `chrome` to
+    use the installed Chrome browser instead of Chromium.
```

Comment:
If I understand correctly, `chromium`, `firefox`, and `webkit` are Playwright's bundled browsers, but when the `chrome` option is used, it relies on the user's installed Chrome instance. If that's correct, could you please update the docstring to clarify this?